plugins/saveactions.c

```diff
@@ -91,11 +93,17 @@ static gchar *config_file;
 
 
 /* Ensures utf8_dir exists and is writable and
- * set backup_dir to the locale encoded form of utf8_dir */
-static gboolean backupcopy_set_backup_dir(const gchar *utf8_dir)
+ * set target to the locale encoded form of utf8_dir */
+static gboolean store_target_directory(const gchar *utf8_dir, gchar **target)
 {
 	gchar *tmp;
 
+	if (target == NULL)
+	{
+		SETPTR(*target, NULL);
```

Comment:
`*target` is guaranteed to dereference a NULL?????

Reply:
This didn't make sense. Thanks for the pointer.
I rewrote the code to return `FALSE` as it should no valid case when the target variable is `NULL`.